youtube_dl/extractor/sverigesradio.py

```diff
@@ -0,0 +1,69 @@
+# coding: utf-8
+from __future__ import unicode_literals
+
+from .common import InfoExtractor
+from ..compat import compat_str
+from ..utils import (
+    try_get,
+    int_or_none,
+)
+
+
+class SverigesRadioBaseIE(InfoExtractor):
+    _GEO_COUNTRIES = ['SE']
+
+    def _extract_audio(self, audio_type, audio_id):
+        query = {'id': audio_id, 'type': audio_type}
+        audiourl = self._download_json('https://sverigesradio.se/sida/playerajax/getaudiourl', audio_id, query=dict(query, quality='high', format='iis'))
+        metadata = self._download_json('https://sverigesradio.se/sida/playerajax/audiometadata', audio_id, query=query)
+        item = metadata['items'][0]
+
+        return {
+            'id': compat_str(audio_id),
+            'url': compat_str(audiourl['audioUrl']),
+            'title': compat_str(item['subtitle']),
+            'series': try_get(item, lambda x: x['title'], compat_str),
+            'duration': int_or_none(item.get('duration')),
+            'thumbnail': try_get(item, lambda x: x['displayimageurl'], compat_str),
+            'description': try_get(item, lambda x: x['description'], compat_str)
+        }
+
+
+class SverigesRadioArtikelIE(SverigesRadioBaseIE):
+    _VALID_URL = r'https?://(?:www\.)?sverigesradio\.se/sida/artikel.aspx\?.*artikel=(?P<id>[0-9]+)'
```

Comment:
Does not match https://sverigesradio.se/sida/gruppsida.aspx?programid=3304&grupp=6247&artikel=7146887.